pkg/chunk/table_manager.go

```diff
@@ -37,11 +43,29 @@ var (
 		Name:      "dynamo_table_capacity_units",
 		Help:      "Per-table DynamoDB capacity, measured in DynamoDB capacity units.",
 	}, []string{"op", "table"})
+	retentionPeriodConfig = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: "cortex",
+		Name:      "retention_period",
+		Help:      "Configured retention period",
+	})
+	tableNumber = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "cortex",
+		Name:      "table_number",
+		Help:      "Min or Max table number for chunks or index kind",
+	}, []string{"extreme", "kind"})
+	tablePeriodConfig = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
How does this metric help with alerting? Would a more useful metric be something for "oldest possible data"?